### PR TITLE
fix(DB/creature): Gregory Victor Gossip Text

### LINKED ISSUE #4458: Doctor Gregory Victor has wrong gossip text  [$5]
##### SMALL DESCRIPTION:
Horde First-Aid Trainer, [Doctor Gregory Victor](https://wotlk.evowow.com/?npc=12920) has the [Alliance trainer](https://wotlk.evowow.com/?npc=12939) dialogue counterpart.

##### CURRENT BEHAVIOUR:
https://cdn.discordapp.com/attachments/806916563810385970/806918959382396968/Wow_JVNhb9gWuz.jpg

##### EXPECTED BLIZZLIKE BEHAVIOUR:
https://wow.gamepedia.com/Doctor_Gregory_Victor

##### STEPS TO REPRODUCE THE PROBLEM:
1. go c id 12920
2. right-click on the npc
3. notice the wrong text

##### AC HASH/COMMIT: 
[adcf3f7a](https://github.com/azerothcore/azerothcore-wotlk/commit/adcf3f7a)

##### DATABASE VERSION: 
WORLD: 2021_02_01_00
AUTH: 2021_01_25_00
CHARACTERS: 2021_01_25_00

##### OPERATING SYSTEM:
Windows 10 - Docker for Desktop

##### EXTRA NOTES: 
Chromie-Client (Downloaded from 10-01-2021)
DBC & Maps - v9 - 1-12-2020

##### MODULES:
None

##### OTHER CUSTOMIZATIONS:
None

<bountysource-plugin>

---
There is a **[$5 open bounty](https://www.bountysource.com/issues/96382377-doctor-gregory-victor-has-wrong-gossip-text?utm_campaign=plugin&utm_content=tracker%2F40032087&utm_medium=issues&utm_source=github)** on this issue. Add to the bounty at [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F40032087&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by Voxstrasza**
## Changes Proposed:
-  Update Gregory Victor's gossip text to be blizzlike
-  Change ID of conflicting entry (not sure where it came from, but that's why the gossip menu didn't work.  Two on one entry)

## Issues Addressed:
- Closes #4458

## Tests Performed:
- Applied SQL and checked entry

## How to Test the Changes
- .go c id 12920
- Read gossip

## Target Branch(es):
- [x] Master

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1613510617294494000.sql [hunk 1]
```diff
@@ -0,0 +1,7 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1613510617294494000');
+
+/* Update Doctor Gregory Victor's gossip to be blizzlike.  Change entry of unknown gossip to avoid conflict.
+*/
+
+UPDATE `creature_template` SET `gossip_menu_id` = 5381 WHERE (`entry` = 12920);
+UPDATE `npc_text` SET `ID`='999999' WHERE  `ID`=6573;
```
Comment (r0m1ntik) [inline data/sql/updates/pending_db_world/rev_1613510617294494000.sql:7]: What for ? this text shows if the player has first aid pumped to 225. No need to change

**Review by r0m1ntik — CHANGES_REQUESTED** [review]

File: data/sql/updates/pending_db_world/rev_1613510617294494000.sql (hunk 1 above)
Comment (r0m1ntik) [reply data/sql/updates/pending_db_world/rev_1613510617294494000.sql:7]: You need to be level 35 and have 225 in First Aid to start the "Triage" quest. I have both. But it simply isnt there. If I interact with the quest giver he will say:

"You have learned all that I am able to teach you at this time, Doctor [Name]. Return to me when your practice has expanded and I shall train you in the ways of the more advanced medicinal arts."

Comment (Voxstrasza) [reply data/sql/updates/pending_db_world/rev_1613510617294494000.sql:7]: I have searched and cannot find proof of this.  The Alliance counterpart does not have this, either.

Comment (r0m1ntik) [reply data/sql/updates/pending_db_world/rev_1613510617294494000.sql:7]: https://fr.classic.wowhead.com/news=294907/classic-wow-artisan-profession-quests-and-trainer-locations#artisan-first-aid-quest-triage

Comment (Voxstrasza) [reply data/sql/updates/pending_db_world/rev_1613510617294494000.sql:7]: Yes, I understand, however I can't find proof that his gossip changes.

Comment (r0m1ntik) [reply data/sql/updates/pending_db_world/rev_1613510617294494000.sql:7]: @Voxstrasza https://www.youtube.com/watch?v=zTvYQ0yqE9w&feature=emb_logo
Look 2:39
Снимок экрана 2021-02-17 в 0 07 07

Comment (Voxstrasza) [reply data/sql/updates/pending_db_world/rev_1613510617294494000.sql:7]: Then I'm not sure how to fix this issue - would SAI be required to update the gossip after the quest is complete?

We would need to add it to the Alliance counterpart as well

**Pull request closed without merging**